src/subscription_manager/certlib.py

```diff
@@ -55,7 +55,10 @@ def log_entry(self):
     def format_exceptions(self):
         buf = ''
         for e in self._exceptions:
-            buf += ' '.join(str(e).split('-')[1:]).strip()
+            if '-' not in str(e):
+                buf += str(e)
+            else:
+                buf += ''.join(str(e).split('-', maxsplit=1)[1:]).strip()
```

Comment:
considering that `split()` always returns a list, which in this case has either 1 (no '-') or 2 elements (`maxsplit=1`), then we can simplify this like this:
```python
    buf += str(e).split('-', maxsplit=1)[-1].strip()
```
no need for checking the character presence, and no need for `join()`